Auth.php

```diff
@@ -1518,6 +1521,29 @@ protected function checkReCaptcha($captcha_response)
         return true;
     }
 
+    /**
+     * Returns a binary format hash of the client's IP address
+     * @return string
+     */
+    private function getIpHash() {
+        return md5($this->getIp() . getenv("PHPAUTH_SECRET"), TRUE);
+    }
+
+    /**
+     * Returns the client's user agent
+     * @return string
+     */
+    private function getUserAgent() {
+        return isset($_SERVER['HTTP_USER_AGENT']) ? $_SERVER['HTTP_USER_AGENT'] : '';
```

Comment:
Here you can wrote:

`return $_SERVER['HTTP_USER_AGENT'] ?? ''`; 

And we can now remove PHP5.6 depencency

Reply:
Great I'll fix this and get rid of PHP5.6 from composer, thanks.